### PR TITLE
Migrate icons to  inline material icon SVGs

### LINKED ISSUE #28: Replace icon-add and icon-delete with material icons
### Is your feature request related to a problem? Please describe.

https://github.com/nextcloud/calendar-availability-vue/search?q=icon

### Describe the solution you'd like

Replace the icon classes with inline material icon SVGs

### Describe alternatives you've considered

N/a

### Additional context

* https://github.com/nextcloud/groupware/issues/38

### PR DESCRIPTION
**Pull request by GretaD**
fixes #28

### CODE REVIEW

File: src/CalendarAvailability.vue [hunk 1]
```diff
@@ -33,23 +37,34 @@
...
 		</template>
 	</div>
 </template>
 
 <script>
-import NcDatetimePicker from '@nextcloud/vue/dist/Components/NcDatetimePicker.js'
+
+import DatetimePicker from '@nextcloud/vue/dist/Components/NcDatetimePicker'
+import Button from '@nextcloud/vue/dist/Components/NcButton'
+import IconAdd from 'vue-material-design-icons/Plus'
```
Comment (ChristophWurst) [inline src/CalendarAvailability.vue:57]: `vue-material-design-icons` is not a dependency of this package at the moment

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Changes in package-lock.json are probably not intentional

File: src/CalendarAvailability.vue (hunk 1 above)
Comment (GretaD) [reply src/CalendarAvailability.vue:57]: ah sorry, you have already stated  inline material icon SVGs on the ticket. Fixing it asap

**Comment by ChristophWurst** [comment]
I'm sorry for the confusion but I did not mean that we vendor the icons. Import them from npm. But add this new dependency to package.json.

File: src/CalendarAvailability.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/CalendarAvailability.vue]: add this dependency

File: package.json [hunk 2]
```diff
@@ -50,7 +50,8 @@
     "rollup-plugin-postcss": "^4.0.1",
     "rollup-plugin-vue": "^5.1.9",
     "vue": "^2.7.8",
-    "vue-template-compiler": "^2.7.8"
+    "vue-template-compiler": "^2.7.8",
+	"vue-material-design-icons": "^5.1.2",
```
Comment (GretaD) [inline package.json:54]: on my php this looks fine, not sure why sometimes is shown like this

Comment (ChristophWurst) [reply package.json:54]: your phpstorm shows tabs as wide as four spaces

the rest of the file is indented with four spaces. your new line uses a tab character.

Comment (ChristophWurst) [reply package.json:54]: this is a production depenednecy, not a dev dependency. move it.

**Comment by GretaD** [comment]
please a review here bitte

File: package.json [hunk 3]
```diff
@@ -50,7 +51,7 @@
     "rollup-plugin-postcss": "^4.0.1",
     "rollup-plugin-vue": "^5.1.9",
     "vue": "^2.7.8",
-    "vue-template-compiler": "^2.7.8"
+    "vue-template-compiler": "^2.7.8",
```
Comment (ChristophWurst) [inline package.json:54]:
```
npm ci
npm ERR! code EJSONPARSE
npm ERR! path /home/christoph/workspace/js/@nextcloud/calendar-availability-vue/package.json
npm ERR! JSON.parse Expected double-quoted property name in JSON at position 1675 while parsing '{
npm ERR! JSON.parse   "name": "@nextcloud/calendar-availab'
npm ERR! JSON.parse Failed to parse JSON data.
npm ERR! JSON.parse Note: package.json must be actual JSON, not just JavaScript.
```

the package-lock.json seems out of sync on main. I'm having a look.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Fix our package* files

```bash
git fetch
git checkout origin/main package*
npm ci
npm i -S vue-material-design-icons
```

**Comment by ChristophWurst** [comment]
> Commit checks / Conventional Commits (pull_request)

When you do your final rebase and squash, change the commit message to `chore: Migrate icon classes to svg icons`

**Comment by GretaD** [comment]
suppressed by #74

**Pull request closed without merging**